application/MainWindow.cpp

```diff
@@ -919,31 +925,52 @@ void MainWindow::updateToolsMenu()
     {
         launchMenu = new QMenu(this);
     }
+    if (launchOfflineMenu) {
+        launchOfflineMenu->clear();
+    }
+    else
+    {
+        launchOfflineMenu = new QMenu(this);
+    }
 
     QAction *normalLaunch = launchMenu->addAction(tr("Launch"));
+    QAction *normalLaunchOffline = launchOfflineMenu->addAction(tr("Launch"));
     connect(normalLaunch, &QAction::triggered, [this]()
             {
-                MMC->launch(m_selectedInstance);
+                MMC->launch(m_selectedInstance, true);
+            });
+    connect(normalLaunchOffline, &QAction::triggered, [this]()
+            {
+                MMC->launch(m_selectedInstance, false);
             });
     launchMenu->addSeparator()->setText(tr("Profilers"));
+    launchOfflineMenu->addSeparator()->setText(tr("Profilers"));
```

Comment:
Ideally, share the same string between the menus. It will make translations less confusing.

Example:
```
QString profilersTitle = tr("Profilers");
launchMenu->addSeparator()->setText(profilersTitle);
launchOfflineMenu->addSeparator()->setText(profilersTitle);
```